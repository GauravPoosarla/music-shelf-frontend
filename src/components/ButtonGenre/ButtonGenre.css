.button-genre button{
  background-color: #4C63AA;
  color: white;
  font-size: 1.2em;
  padding-top: 5px;
  padding-bottom: 5px;
  border-radius: 20px;
  border: none;
  text-align: center;
  font-weight: bold;
  margin-top: 50px;
  padding-left: 50px;
  padding-right: 50px;
  position: absolute;
  right: 70%;
}

@media screen and (max-width: 250px) {
  .button-genre button{
    right: 10%;
  }
}

@media screen and  (min-width: 251px) and (max-width: 300px) {
  .button-genre button{
    right: 30%;
  }
}

@media screen and  (min-width: 301px) and (max-width: 500px){
  .button-genre button{
    right: 50%;
  }
}

@media screen and (min-width: 501px) and (max-width: 800px) {
  .button-genre button{
    right: 50%;
  }
}

@media screen and (min-width: 8001px) and (max-width: 1000px) {
  .button-genre button{
    right: 55%;
  }
}