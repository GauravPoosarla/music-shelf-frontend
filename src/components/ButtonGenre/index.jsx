import React from 'react';
import './ButtonGenre.css';

const ButtonGenre = (props) => {
  return (
    <div className="button-genre">
      <button>{props.genre}</button>
    </div>
  );
};

export default ButtonGenre;
