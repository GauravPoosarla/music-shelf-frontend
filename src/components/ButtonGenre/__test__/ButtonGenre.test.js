import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import ButtonGenre from '..';

describe('ButtonGenre', () => {
  it('should render correctly with props', () => {
    const { container } = render(<ButtonGenre genre="Country" />);
    const button = container.querySelector('.button-genre');
    expect(button.textContent).toBe('Country');
    expect(container).toMatchSnapshot();
  });
});
