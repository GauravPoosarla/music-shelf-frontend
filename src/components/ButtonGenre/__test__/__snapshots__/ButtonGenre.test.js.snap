// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGenre should render correctly with props 1`] = `
<div>
  <div
    class="button-genre"
  >
    <button>
      Country
    </button>
  </div>
</div>
`;
